samples/generic-base/logging/user-values-osd.yaml

```diff
@@ -1,12 +1,11 @@
-# Chart: https://github.com/opensearch-project/helm-charts/tree/main/charts/opensearch-dashboards
```

Comment:
How do you feel about including the version-specific link to the charts/values files?  Part of me would like to keep these links "generic" but given that our code is using a specific version, it seems like we should point them to that version.  On the other hand, this results in another place we need to update Helm chart version numbers...which complicates maintenance.  Thoughts?